README.md

```diff
@@ -226,6 +226,9 @@ Changes to X or Y force recomputation of all results when a sample is rerun usin
 
 When releasing a new version, please add a Git tag of the form `vX.Y.Z`.
 
+- 4.3.2
```

Comment:
Would you mind doing 4.3.3 ? I was planning to push a change soon.

Reply:
Actually go ahead. this is a very innocuous change, so lets do this one first.

Reply:
Ah okay, I'll leave it at 4.3.2.